### PR TITLE
[MM-46596] Screen Reader Announces Login Successful

### PR DESCRIPTION
**Pull request by AbhinavVihan**
#### Summary
Screen Reader Announces Login Successfull

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-46596

#### Related Pull Requests
None

#### Screenshots
before   ---- 

https://user-images.githubusercontent.com/89139362/188879053-1c5baeff-901b-46ee-91f6-6a8da874e436.mp4

after ---  

https://user-images.githubusercontent.com/89139362/190139597-502f4feb-ccc7-4c2c-a8dc-ea8f46c0a270.mp4

#### Release Note

```release-note
   Screen Reader Announces Login Successfull

```

### CODE REVIEW

File: components/login/login.tsx [hunk 1]
```diff
@@ -613,6 +628,7 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
         // Record a successful login to local storage. If an unintentional logout occurs, e.g.
         // via session expiration, this bit won't get reset and we can notify the user as such.
         LocalStorageStore.setWasLoggedIn(true);
+        srSpeak('Login Successfull', 'assertive');
```
Comment (AshishDhama) [inline components/login/login.tsx:631]: @AbhinavVihan use FormatMessage to localise message.

**Comment by AshishDhama** [comment]
@nevyangelova I think you have an idea of accessibility, if you could confirm, it is the right approach for this?

**Review by nevyangelova — CHANGES_REQUESTED** [review]
Thanks @AbhinavVihan I would like to suggest a different approach. You do not need to add a function for this, as HTML5 elements support their own accessibility props. In this case you can check which elements are being read out loud when login is successful and remove them from the login group, as well as adding missing aria-labels. You can read about it here https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label
https://developer.mozilla.org/en-US/docs/Web/Accessibility

**Review by nevyangelova — APPROVED** [review]
Thanks, looks great!

**Review by AshishDhama — APPROVED** [review]

**Comment by AshishDhama** [comment]
@AbhinavVihan Could you please update the failing snapshot

**Comment by AbhinavVihan** [comment]
on it

**Review by AshishDhama — APPROVED** [review]

**Comment by jgilliam17** [comment]
Thanks @AbhinavVihan 
The announcement at login is "Mattermost message input complementary region" since that is where the focus is placed at login. No "login successful" announcement after user presses Enter to log in.
Tested using Jaws reader on Windows.

**Comment by AbhinavVihan** [comment]
@jgilliam17, Would you take a look please

**Comment by jgilliam17** [comment]
Thank you for the quick fix @AbhinavVihan 
When testing manually, reader is announcing "login successful".
[E2E report](https://automation-dashboard.vercel.app/cycles/c1e929d9-070a-4e47-83c1-54213b2517a0) show that 2 tests are failing in tests/integration/accessibility/accessibility_nav_diff_regions_spec.js
Can you please take a look?

**Comment by AbhinavVihan** [comment]
Hi @jgilliam17, I don't think there was some failing issue in the tests, I've rechecked it.
Screenshot from 2022-10-01 08-50-19

**Review by jgilliam17 — APPROVED** [review]
Thanks for verifying the tests @AbhinavVihan. 
Looks good to merge.
@AshishDhama can you please approve the last workflow and merge - thanks 🙂

**Comment by amyblais** [comment]
@nevyangelova @AshishDhama Can you help manually cherry-pick this to the ``release-7.1`` branch? I was requested to backport this to the v7.1 ESR.

**Pull request merged**